### ISSUE TITLE
GoAccess debian repository install does not work on Stretch (libssl1.0.2)

### ISSUE THREAD

**Issue by MacGyver27** [allinurl/goaccess#1592]
same as #716 
```
apt-get install goaccess
Reading package lists... Done
Building dependency tree
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 goaccess : Depends: libssl1.0.2 (>= 1.0.2) but it is not installable
E: Unable to correct problems, you have held broken packages.
```

**Comment by aremai** [comment]
i've been following the official guide here:
https://goaccess.io/download#official-repo

**Comment by allinurl** [comment]
v1.4 will support Buster. Shooting for the end of March, early Apr.

**Comment by whalehub** [comment]
@allinurl Any update on the ETA for 1.4?

**Comment by allinurl** [comment]
@whalehub Yesterday I was able to finish a big chunk of refactoring related to the storage engine. That should align things pretty well for a very soon v1.4. My goal was to release v1.4 on mid Apr, though I don't expect it to go farther than mid May. 

Since testing plays a big part of the release, if you are willing to give a hand, please feel free to test out the [**development**](https://github.com/allinurl/goaccess#build-from-github-development) version and let me know how things go on your end :)

**Comment by szepeviktor** [comment]
> a very soon v1.4

@allinurl Could we have a COVID-19 release?

**Comment by allinurl** [comment]
@szepeviktor sounds doable, working on it :)

**Comment by allinurl** [comment]
This has been fixed and it's now available in [**v1.4**](https://goaccess.io/download). Thank you!
